Documentation/English/EXOTIC Installation Instructions for Windows 10.md

```diff
@@ -0,0 +1,93 @@
+# EXOTIC Installation Instructions for Windows 10
+
+## Running EXOTIC
+There are two ways you can run EXOTIC: firstly through a web browser interface with a so-called Python Notebook running in Google Colab or in a Jupyter Notebook running locally on your own personal computer. 
+
+Alternatively, you can install both Python and the EXOTIC program to your own computer and run them there. The program uses a combination of a graphical front-end to enter values and a command line window where you can view progress as the program runs.
+
+Read the details of these two approaches below and select the best option for you:
+
+### Google Colab - *RECOMMENDED*:
```

Comment:
Can you remove "*RECOMMENDED*" here? I am not sure that's still the case now. :)

Reply:
Done and subsequent text amended to reflect change of status.